x-pack/plugins/lens/public/app_plugin/csv_download_provider/csv_download_provider.tsx

```diff
@@ -143,28 +141,23 @@ export const downloadCsvShareProvider = ({
 
     return [
       {
-        ...menuItemMetadata,
+        shareMenuItem: {
+          name: panelTitle,
+          icon: 'document',
+          disabled: !csvEnabled,
+          sortOrder: 1,
+        },
         label: 'CSV' as const,
         reportType: 'lens_csv' as const,
         generateExport: downloadCSVHandler,
+        warnings: getWarnings(activeData),
         ...(atLeastGold()
           ? {
-              helpText: (
-                <FormattedMessage
-                  id="xpack.lens.share.helpText"
-                  defaultMessage="Export a CSV of this visualization."
```

Comment:
This unique help text is probably worth keeping, right? Assuming that bringing this back will make it override the default.

Reply:
I removed this after noticing that the `helpText` is updated based on what option the user selects. For the **PNG** and **PDF** options this text is generic and prompts the user to select the desired file type. But when you select the csv option, it now explains that specific option, see demo below.

![Zight Recording 2024-10-20 at 07 18 29 PM](https://github.com/user-attachments/assets/96196684-c9f2-46a9-8c96-ee2914548a2a)

This difference is strange to me. I feel we should always be generic or always be specific. So I chose to be generic as it also prompts the user on what action to take whereas the specific is describing each option as it's selected. We could be generic on top then add the case-specific help text based on the selected option but that seems a bit much as the options are pretty obvious and don't require much explanation. To that point the old csv specific text just said _Export a CSV of this visualization_.

But this does come in handy when there is only one option to select in which case the `helpText` can be altered as needed.

Reply:
All good! I just wanted to know your thoughts about my observation, in case might have led to something useful. I will approve the PR to unblock you.